packages/chart/src/Pie.ts

```diff
@@ -436,14 +438,17 @@ export interface Pie {
     labelHeight(_: number): this;
     seriesPercentageFormat(): string;
     seriesPercentageFormat(_: string): this;
+    showLabels(): boolean;
+    showLabels(_: boolean): this;
 }
-Pie.prototype.publish("showSeriesValue", false, "boolean", "Append data series value next to label");
+Pie.prototype.publish("showLabels", true, "boolean", "If true, wedge labels will display");
+Pie.prototype.publish("showSeriesValue", false, "boolean", "Append data series value next to label", null, { disable: w => !w.showLabels() });
 Pie.prototype.publish("seriesValueFormat", ",.0f", "string", "Number format used for formatting series values", null, { disable: w => !w.showSeriesValue() });
-Pie.prototype.publish("showSeriesPercentage", false, "boolean", "Append data series percentage next to label");
+Pie.prototype.publish("showSeriesPercentage", false, "boolean", "Append data series percentage next to label", null, { disable: w => !w.showLabels() });
 Pie.prototype.publish("seriesPercentageFormat", ",.0f", "string", "Number format used for formatting series percentages", null, { disable: w => !w.showSeriesPercentage() });
```

Comment:
@GordonSmith I assume it's fine not to add `!w.showLabels()` to this disable condition... because `w.showSeriesPercentage()` will default to false if `w.showLabels()` is false